### ISSUE TITLE
Binaries for mixed arch fleet

### ISSUE THREAD

**Issue by blmhemu** [nahsi/ansible-nomad#62]
In https://github.com/nahsi/ansible-nomad/blob/9cc8fd6f75f684cb3f6529f87267c0add2c32add/tasks/01-install.yml#L3
We download the binary only once. But what if there are machines with multiple arch in the fleet (say amd64, arm65, arm). What happens in such cases ?

**Comment by nahsi** [comment]
Yes, this is a bug. Binary will be downloaded only once. Thanks for bringing that up, I will fix it.
